l5kit/l5kit/geometry/transform.py

```diff
@@ -1,10 +1,31 @@
-from typing import Optional, Sequence, Tuple, Union, cast
+from typing import Sequence, Union, cast
 
 import numpy as np
 import pymap3d as pm
 import transforms3d
 
 
+def agent_pose(agent_centroid_m: np.ndarray, agent_yaw_rad: float) -> np.ndarray:
```

Comment:
Any reason why this is called agent_pose and not in style of 'world_to_image_pixels_matrix'?

Reply:
this is at the same time the `agent_pose`  and the `world_from_agent` matrix, I couldn’t come up with a better name :( @yye-lyft ?

Reply:
`agent_pose_world` == `world_from_agent`
Maybe we can rename this function to `compute_agent_pose` if you want. @lucabergamini